### PR TITLE
Problem(CRO-549) client withdraw all doesn't use dummy values in fee estimation

### PR DESCRIPTION
**Pull request by linfeng-crypto**
solution: Mock a signature to calculate fee.

### CODE REVIEW

File: client-network/src/network_ops/default_network_ops_client.rs [hunk 1]
```diff
@@ -91,6 +91,35 @@ where
...
+        let ecdsa_signature =
+            RecoverableSignature::from_compact(&[0; 64], RecoveryId::from_i32(1).unwrap()).unwrap();
+        let signature = StakedStateOpWitness::new(ecdsa_signature);
+
+        let signed_transaction = SignedTransaction::WithdrawUnbondedStakeTransaction(
+            transaction,
+            staked_state,
+            signature,
+        );
+        let tx_aux = self.transaction_cipher.encrypt(signed_transaction)?;
```
[the remark this answers is not in the document]
Comment (linfeng-crypto) [reply client-network/src/network_ops/default_network_ops_client.rs]: should the expayload padded by 128bits?

[the remark this answers is not in the document]
Comment (tomtau) [reply client-network/src/network_ops/default_network_ops_client.rs]: yeah, same

[the remark this answers is not in the document]
Comment (tomtau) [reply client-network/src/network_ops/default_network_ops_client.rs]: you may extract / reuse that dummy signer functionality

**Pull request merged**